cogs/welcome/welcome.py

```diff
@@ -28,8 +31,23 @@ class Welcome(commands.Cog):  # pylint: disable=too-many-instance-attributes
     def __init__(self, bot: Red):
         self.bot = bot
         self.config = Config.get_conf(self, identifier=5842647, force_registration=True)
+
         self.config.register_guild(**DEFAULT_GUILD)
 
+        self.dataDir = data_manager.cog_data_path(cog_instance=self)
+        self.imgDir = self.dataDir / WELCOME_IMG_FOLDER
+        self.bundled_assets = data_manager.bundled_data_path(self)
+
+        # create folder to hold welcome images
+        try:
+            os.makedirs(self.imgDir, exist_ok=True)
```

Comment:
Since `self.imgDir` is a `pathlib.Path`, can we also opt for `pathlib.Path` instead of `os.*`?